controller/ampachecontroller.php

```diff
@@ -501,4 +502,26 @@ protected function get_cover() {
 			return $r;
 		}
 	}
+
+	/**
+	 * this is not a part of ampache specification
+	 * This is an exposed API endpoint.
+	 * Client uses settings#addUserKey instead
+	 * @NoAdminRequired
+	 * @NoCSRFRequired
+	 * @CORS
```

Comment:
I tried to do the same initially but turns out an action annotated with `@CORS` is not allowed to be used in conjunction with session authentication https://github.com/owncloud/core/blob/master/lib/private/appframework/middleware/security/corsmiddleware.php#L82